build-tests/api-documenter-scenarios/tsdoc-metadata.json

```diff
@@ -0,0 +1,11 @@
+// This file is read by tools that parse documentation comments conforming to the TSDoc standard.
+// It should be published with your NPM package.  It should not be tracked by Git.
```

Comment:
This file is not supposed to be tracked by Git.  Normally it gets written to a folder such as `dist` that will get ignored by `.gitignore`.

The output folder is controlled [by complicated heuristics](https://api-extractor.com/pages/configs/api-extractor_json/#tsdocmetadatatsdocmetadatafilepath) (see implementation [here](https://github.com/microsoft/rushstack/blob/c7aef037ee57dbb0892b2882c9d8404cc51ce0fc/apps/api-extractor/src/analyzer/PackageMetadataManager.ts#L83)).  In practice however, it is almost always determined by the `"typings"` or `"index"` field of **package.json**.  When it gets accidentally committed to Git, it is usually because that was not set up yet when API Extractor was first run.

Reply:
Ah, it was added because I omitted the `"typings"` field from the `package.json`. Should be fixed now.